provider/cmd/pulumi-resource-aws/bridge-metadata.json

```diff
@@ -232628,4 +232628,4 @@
             "aws:workspaces/getWorkspace:getWorkspace": 0
         }
     }
-}
+}
```

Comment:
I'm not quite sure why this newline started disappearing. I tried adding it back and regenerating the file a couple of times but it kept happening